tracee-ebpf/main.go

```diff
@@ -101,9 +101,17 @@ func main() {
 				return err
 			}
 
-			missingKernelOptions := missingKernelConfigOptions()
-			if len(missingKernelOptions) != 0 {
-				return fmt.Errorf("kernel is not properly configured, missing: %v", missingKernelOptions)
+			if kernelConfig := helpers.InitKernelConfig(); kernelConfig == nil {
+				return err
```

Comment:
This will fail tracee-ebpf in cases that we couldn't get the kconfig for some reason, while it is possible that we are still able to run tracee-ebpf correctly. Also see my comment below